apps/brew-careers/util/createHtmlElement/createHtmlElement.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable @typescript-eslint/consistent-type-imports */
+import { Block, RichText } from "./createHtmlElement.types";
+
+import React from "react";
+import createStyleString from "../createStyleString/createStyleString";
+
+function createHtmlElement(block: Block): React.ReactNode {
+  let elementType: string;
+  let content: React.ReactNode[] = [];
+
+  switch (block.type) {
```

Comment:
You can use object instead of switch case, like this;
In this way, you map it with a single object instead of many cases.

```
const types={
"heading_1": "h2",
"heading_2":"h3",
...
}

elemetType=types[block.type]
if(!elementType){
elementType="div"
}
```